R/fetch_distribution_history.R

```diff
@@ -15,7 +16,10 @@
 #'
 #' surveys <- all_surveys()
 #' distributions <- fetch_distributions(surveys$id[1])
-#' distribution_history <- fetch_distribution_history(distributions$id[1])
+#' distribution_history <- fetch_distribution_history(
+#'     distributions$id[1],
+#'     surveyID = surveys$id[1]
+#' )
```

Comment:
Is this correct? The `surveyID` arg is required, right? (I will fix them to match style-wise in the next commit, with both not needing names.)

Reply:
Correct, surveyID is required if you want emails associated with the distribution list and not just contact ids. The survey id is necessary for `list_distribution_links` embedded in `fetch_distribution_history`. Feel free to make any style changes you see necessary.

@chrisumphlett may be able to help with the formal testing given that he has an XM directory license.